zbus/src/address/transport/systemd.rs

```diff
@@ -0,0 +1,26 @@
+use std::marker::PhantomData;
+
+use super::{Address, Error, KeyValFmt, KeyValFmtAdd, Result};
+
+/// `systemd:` D-Bus transport.
```

Comment:
Could you please elaborate just a little what this is? Link to some doc would be nice.

Reply:
added link to spec